apps/frontend/src/app/access-portal/pages/home/components/post-card/post-card.component.html

```diff
@@ -1,11 +1,16 @@
 <div class="border-neutral-6 flex flex-col rounded-sm border-4 p-2 sm:p-7">
   <div class="flex justify-between">
     <div class="flex items-center">
-      <img
-        [src]="sanitizeUrl(publication.profile.picture?.original?.url)"
-        alt="avatar example"
-        class="h-12 w-12 rounded-full object-cover"
-      />
+      <button
+        [routerLink]="['/home/profile-public']"
+        [queryParams]="{ id: publication.profile.id }"
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/79bc28d7-7c9a-4f3f-a50b-bedb13c52d6d)

there is something weird with the layout, part of if is not selectable. also the date is a little squashed.